src/epanet.c

```diff
@@ -2831,22 +2866,22 @@ int DLLEXPORT EN_getbasedemand(EN_Project p, int nodeIndex, int demandIndex,
 **----------------------------------------------------------------
```

Comment:
I think we should deprecate these demand property access functions as discussed in #467.